.github/workflows/e2e-parallel-full.yml

```diff
@@ -125,9 +126,10 @@ jobs:
           export AWS_CSM_ENABLED=true
           export AWS_CSM_PORT=31000
           export AWS_CSM_HOST=127.0.0.1
-          terraform destroy -target=module.eks_blueprints_kubernetes_addons  -no-color -input=false -auto-approve
-          terraform destroy -target=module.eks_blueprints  -no-color -input=false -auto-approve
-          terraform destroy -target=module.eks  -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks_blueprints_kubernetes_addons -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks_blueprints_addons -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks_blueprints -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks -no-color -input=false -auto-approve
```

Comment:
And here.

Reply:
more correct now because of this `-target=module.eks_blueprints_addons`